docs/dm/zh_CN/maintenance/query-status.md

```diff
@@ -0,0 +1,130 @@
+query-status
+===
+
+### 索引
+- [查询结果介绍](#查询结果介绍)
+- [子任务状态](#子任务状态)
+
+### 查询结果介绍
+
+```
+» query-status
+{
+    "result": true,     # 表示查询是否成功
+    "msg": "",          # 查询失败时，失败原因的描述
+    "workers": [                            # DM-worker 列表
+        {
+            "result": true,
+            "worker": "172.17.0.2:10081",   # 该 DM-worker 的 host:port 信息
+            "msg": "",
+            "subTaskStatus": [              # 该 DM-worker 所有子任务信息
+                {
+                    "name": "test",         # 任务名称
+                    "stage": "Running",     # 子任务运行状态，包含 New, Running, Paused, Stopped, Finished。不同状态的含义和状态转换关系请参考[子任务状态]
+                    "unit": "Sync",         # DM 处理单元，包括 Check, Dump, Load, Sync
+                    "result": null,
+                    "unresolvedDDLLockID": "",  # sharding DDL 锁ID
+                    "sync": {                   # sync 处理单元同步信息，总是显示与当前处理单元相同的组件同步信息
+                        "totalEvents": "12",    # 该子任务同步的总的 binlog 事件数
+                        "totalTps": "1",        # 该子任务每秒同步 binlog 事件数
+                        "recentTps": "1",       # 该子任务最近1秒同步的 binlog 事件数
+                        "masterBinlog": "(bin.000001, 3234)",                               # 上游数据库 binlog position
+                        "masterBinlogGtid": "c0149e17-dff1-11e8-b6a8-0242ac110004:1-14",    # 上游数据库 GTID 信息
+                        "syncerBinlog": "(bin.000001, 2525)",                               # sync 处理单元已经同步的 binlog position
+                        "syncerBinlogGtid": "",                                             # 目前该值始终为空
+                        "blockingDDLs": [       # 当前被阻塞的DDL列表
+                            "USE `test`; ALTER TABLE `test`.`t_target` DROP COLUMN `age`;"
```

Comment:
this will be empty when `unsynced` is not empty, right? any description?

Reply:
yes, added some description